### PR TITLE
gcalbot: select improvements

### PR DESCRIPTION
**Pull request by marceloneil**
Screen Shot 2020-03-02 at 4 02 45 PM

### CODE REVIEW

File: gcalbot/gcalbot/html.go [hunk 1]
```diff
@@ -146,6 +136,93 @@ const tmplHeader = `<!DOCTYPE html>
...
+	//	outline: 0;
+	//}
+	//
+	//.checkbox-container input:checked {
+	//	background-color: rgb(76, 142, 255);
+	//	border-color: rgb(76, 142, 255);
+	//	border-radius: 2px;
+	//	border-style: solid;
+	//	border-width: 1px;
+	//}
```
Comment (hyperobject) [inline gcalbot/gcalbot/html.go]: maybe remove these styles if they're not being used?

File: gcalbot/gcalbot/html.go [hunk 2]
```diff
@@ -325,29 +423,50 @@ const tmplConfig = `{{template "header" .}}
...
-			{{end}}
-		</select>
+		<div class="select-container">
+			<select name="ds_time">
+				{{range .DSTimeOptions}}
+					<option value="{{.Minute}}" {{if eq .Minute $.DSTime}} selected {{end}}>{{.Title}}</option>
+				{{end}}
+			</select>
+			<div class="caret">
+				<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
```
Comment (hyperobject) [inline gcalbot/gcalbot/html.go]: would be nice to not have to hardcode the same svg a bunch of times - maybe put this in a variable or with the other image?

File: gcalbot/gcalbot/html.go (hunk 1 above)
Comment (marceloneil) [reply gcalbot/gcalbot/html.go]: Will do. However, do you think that the button style should be changed? cc @joshblum

Comment (joshblum) [reply gcalbot/gcalbot/html.go]: woudl be nice if the button style was consistent across platforms esp mobile

**Comment by marceloneil** [comment]
@hyperobject I think I'll do the button styling in a future pr... is this good to go in?

**Comment by hyperobject** [comment]
> @hyperobject I think I'll do the button styling in a future pr... is this good to go in?
one thing i missed - would be good to make the caret a grey color if the dropdown is disabled (and maybe the background too?)

**Pull request merged**